[2025-10-03T12:57:11.525+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-03T12:57:11.553+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_four.insert_row_task scheduled__2025-10-03T12:54:00+00:00 [queued]>
[2025-10-03T12:57:11.562+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_four.insert_row_task scheduled__2025-10-03T12:54:00+00:00 [queued]>
[2025-10-03T12:57:11.563+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-03T12:57:11.585+0000] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): insert_row_task> on 2025-10-03 12:54:00+00:00
[2025-10-03T12:57:11.594+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=572) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-03T12:57:11.596+0000] {standard_task_runner.py:63} INFO - Started process 574 to run task
[2025-10-03T12:57:11.596+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_four', 'insert_row_task', 'scheduled__2025-10-03T12:54:00+00:00', '--job-id', '140', '--raw', '--subdir', 'DAGS_FOLDER/dag_four.py', '--cfg-path', '/tmp/tmpdeytokki']
[2025-10-03T12:57:11.601+0000] {standard_task_runner.py:91} INFO - Job 140: Subtask insert_row_task
[2025-10-03T12:57:11.621+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-10-03T12:57:11.694+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_four.insert_row_task scheduled__2025-10-03T12:54:00+00:00 [running]> on host 7aff8d0a020c
[2025-10-03T12:57:11.834+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_four' AIRFLOW_CTX_TASK_ID='insert_row_task' AIRFLOW_CTX_EXECUTION_DATE='2025-10-03T12:54:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-03T12:54:00+00:00'
[2025-10-03T12:57:11.837+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-03T12:57:11.843+0000] {sql.py:276} INFO - Executing: INSERT INTO my_table (name) VALUES ('Hello from Airflow');
[2025-10-03T12:57:11.855+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2025-10-03T12:57:11.870+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2025-10-03T12:57:11.876+0000] {sql.py:487} INFO - Running statement: INSERT INTO my_table (name) VALUES ('Hello from Airflow');, parameters: None
[2025-10-03T12:57:11.880+0000] {sql.py:496} INFO - Rows affected: 1
[2025-10-03T12:57:11.888+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-03T12:57:11.905+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_four, task_id=insert_row_task, run_id=scheduled__2025-10-03T12:54:00+00:00, execution_date=20251003T125400, start_date=20251003T125711, end_date=20251003T125711
[2025-10-03T12:57:11.974+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-03T12:57:12.026+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/models/baseoperator.py:1297: AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
  result = cls.__new__(cls)

[2025-10-03T12:57:12.121+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-03T12:57:12.150+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
